src/components/Fields/TextArea.story.js

```diff
@@ -34,13 +34,13 @@ stories.add(
 );
 
 stories.add(
-    'Label Always',
+    'Auto Hide Label',
     storyWrapper(
-        'Label can be forced to always appear with the hasLabelAlways prop.',
+        'Use `autoHideLabel` to hide the field label when the input loses focus.',
         <TextArea
             label="Always has a label"
```

Comment:
This could probably do with a better example label since it doesn't always have a visible label.

Btw, the label is always visible when value is empty, should we mention that in the description?

Reply:
I've worked this to give far better examples